.github/CODEOWNERS

```
@@ -1 +1,2 @@
 * @exercism/maintainers-admin
+* neenjaw
```

The second line will override the first. What you really want is to make you (and I think angelika so you can review each other's stuff) owners of everything, then leave the maintainers admin to be codeowners of this file.

You also need the `@` for your name

```suggestion
* @neenjaw @angelikatyborska 
.github/CODEOWNERS @exercism/maintainers-admin
```

Could you update the other repos accordingly pls? (cc @angelikatyborska)

---

Oops! Yes

---

Thank you for adding me to codeowners of the three Elixir tools! ❤️